templates/committee/_layout.html.twig

```diff
@@ -2,6 +2,10 @@
 
 {% set is_host = is_host(committee) %}
 
+{%- block meta_description -%}
+    {{ committee.name }} est un comité local de La République En Marche ! créé par {{ committee_hosts[0].partialName }} et situé dans la ville de {{ committee.cityName }}. Il est composé de {{ 'committee.members_count'|transchoice(committee.membersCount) }} adhérents actifs. Les comités locaux sont le fondement de notre mouvement, rejoignez-en un pour participer au renouvellement de la politique française !
```

Comment:
Is it possible to add some new lines here ? Or it would not fit the meta_description content ?